src/debugger/disasm.c

```diff
@@ -0,0 +1,375 @@
+#include <PR/ultratypes.h>
+#include <stdio.h>
+
+#include "sm64.h"
+#include "macros.h"
+#include "farcall.h"
+#include "disasm.h"
+#include "map_parser.h"
+
+static char insn_as_string[100];
+
+InsnTemplate insn_db[] = {
```

Comment:
Is it possible to bring all this code with map file? Seems like it wastes useful runtime space